src/platform/silabs/multi-ota/OTAFactoryDataProcessor.cpp

```diff
@@ -17,37 +17,20 @@
  */
 
 #include <lib/core/TLV.h>
-#include <platform/internal/CHIPDeviceLayerInternal.h>
 #include <platform/silabs/multi-ota/OTAFactoryDataProcessor.h>
 
 namespace chip {
 
 using namespace ::chip::DeviceLayer::Silabs;
 
-CHIP_ERROR OTAFactoryDataProcessor::Init()
-{
-    mAccumulator.Init(mLength);
-
-    return CHIP_NO_ERROR;
-}
-
-CHIP_ERROR OTAFactoryDataProcessor::Clear()
-{
-    OTATlvProcessor::ClearInternal();
-    mAccumulator.Clear();
-    mPayload.Clear();
-
-    return CHIP_NO_ERROR;
-}
-
 CHIP_ERROR OTAFactoryDataProcessor::ProcessInternal(ByteSpan & block)
 {
     CHIP_ERROR error = CHIP_NO_ERROR;
 
     ReturnErrorOnFailure(mAccumulator.Accumulate(block));
 #if OTA_ENCRYPTION_ENABLE
-    MutableByteSpan mBlock = MutableByteSpan(mAccumulator.data(), mAccumulator.GetThreshold());
-    OTATlvProcessor::vOtaProcessInternalEncryption(mBlock);
+    MutableByteSpan byteblock = MutableByteSpan(mAccumulator.data(), mAccumulator.GetThreshold());
```

Comment:
nit styling shall be respected. Variables name use lowerCamelCase
```suggestion
    MutableByteSpan byteBlock = MutableByteSpan(mAccumulator.data(), mAccumulator.GetThreshold());
```